src/Microsoft.Data.SqlClient/tests/FunctionalTests/SqlCommandTest.cs

```diff
@@ -518,5 +520,110 @@ public void ParameterCollectionTest()
                 cmd.Parameters.Remove(cmd.Parameters[0]);
             }
         }
+
+        #region fixing "Change SqlParameter to allow empty IEnumerable<SqlDataRecord>? #2971"
```

Comment:
I'm not sure what to do about these tests. I think someone with a bit more experience in the test area could provide more concrete feedback. @mdaigle ?